src/features/vscode/messages.ts

```diff
@@ -0,0 +1,58 @@
+// This file also needs to be copied to source-academy/frontend
```

Comment:
Maybe for the one committed to the frontend, you can change the comment to point to the original file in the extension repo?